apps/docs/tsconfig.json

```diff
@@ -25,6 +25,6 @@
     "**/*.tsx",
     ".next/types/**/*.ts",
     ".contentlayer/generated"
-  ],
+, "content/docs/components/alert.mdx"  ],
```

Comment:
what's that for?

Reply:
It appeared automatically when I ran pnpm build for working with docs